lib/experimental-default-theme.json

```diff
@@ -130,6 +130,9 @@
 			},
 			"color": {
 				"custom": true
+			},
```

Comment:
Food for thought: I'm finding the current format too verbose.

```json
{
  "features": {
    "typography": {
      "dropCap": true
    },
    "color": {
      "custom": true
    },
    "gradient": {
      "custom": true
    }
  }
}
```

My expectations were that we'd group the features under `typography` and `color` sub-families, as we do in the style subtree. If we aren't going to group the features _by family_ I think I'd prefer something a bit more terse, along these lines:

```json
{
  "features": {
      "dropCap": true,
      "customColor": true,
      "customGradient": true
  }
}
```

Reply:
I agree that at the current state, it doesn't make sense to have the nesting but I also think we'll probably add things under the top level keys. In fact as you see on my linked comment above, I initially was thinking the preset is one of these keys (since the preset also allows to disable the control when you provide an empty one).

I'm not too concerned at the moment but this is something we should revisit once we move all the features.